packages/amplify-e2e-tests/src/__tests__/rds-model-v2.test.ts

```diff
@@ -616,7 +647,7 @@ describe('RDS Model Directive', () => {
     return listResult;
   };
 
-  const checkGenericError = async (errorMessage?: string) => {
+  const checkGenericError = async (errorMessage?: string): Promise<void> => {
```

Comment:
Why was this async?

Reply:
Oops, it doesn't have to be. This is an existing code, caught by lint fix.